h/views/client.py

```diff
@@ -125,11 +162,18 @@ def absolute_asset_urls(bundle_name):
         return [urlparse.urljoin(base_url, url)
                 for url in assets_env.urls(bundle_name)]
 
-    return {
-        'app_html_url': request.route_url('sidebar_app'),
-        'inject_resource_urls': (absolute_asset_urls('inject_js') +
-                                 absolute_asset_urls('inject_css'))
-    }
+    client_boot_url = _resolve_client_boot_url(request)
```

Comment:
Rather than adding complexity to this view function, it probably makes more sense to have a separate view function which uses the `has_feature_flag` view predicate:

```python
from pyramid.config import not_

...

@view_config(route_name='embed',
             renderer='h:templates/embed.js.jinja2',
             has_feature_flag=not_('use_client_boot_script'))
def embed(request):
    # current view

@view_config(route_name='embed',
             has_feature_flag='use_client_boot_script')
def embed_redirect(request):
    # redirect
```